main.tf

```
@@ -0,0 +1,31 @@
+
+locals {
+  enforce_uppercase = lower(var.enforce_case) == "upper"
+  enforce_lowercase = lower(var.enforce_case) == "lower"
+
+  uncased_name = join(var.name_seperator, var.names)
+  name         = local.enforce_uppercase ? upper(local.uncased_name) : local.enforce_lowercase ? lower(local.uncased_name) : local.uncased_name
```

let's break up this long line

```suggestion
  name = (
    local.enforce_uppercase ?
      upper(local.uncased_name) :
      local.enforce_lowercase ?
        lower(local.uncased_name) :
        local.uncased_name
  )
```

---

This is more readable. But isn't code that's hard to write supposed to be hard to read?